<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_Blend</class>
 <widget class="QDialog" name="Dialog_Blend">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1142</width>
    <height>259</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="Image_Fundo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>541</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="Image_Processada">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>541</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>541</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QTextBrowser" name="Output_txt">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>10</y>
     <width>541</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Valor_Slider">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>210</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="Botao_Blend">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>190</y>
     <width>83</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Misturar</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="Botao_Sobrepor">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>190</y>
     <width>83</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Soprepor</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Botao_Usar">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>190</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Usar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Botao_Cancel">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>190</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Imagem de ReferÃªncia:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>90</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Imagem de Trabalho:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
